ui/dashboards/src/components/Panel/Panel.tsx

```diff
@@ -23,13 +23,18 @@ import { PanelContent } from './PanelContent';
 export interface PanelProps extends CardProps<'section'> {
   definition: PanelDefinition;
   editHandlers?: PanelHeaderProps['editHandlers'];
+  panelOptions?: PanelOptions;
 }
 
+export type PanelOptions = {
+  extra?: () => React.ReactNode;
```

Comment:
Named this prop `extra` which was inspired by Ant Design's extra prop [[1]](https://ant.design/components/card#api) [[2]](https://ant.design/components/drawer#api) they add to their components
<img width="1339" alt="image" src="https://github.com/perses/perses/assets/6935733/0d90f153-2bd2-48df-bdf5-13165127d651">

However, I'm open to another name but figured I copy a popular component library paradigm.

Reply:
I could see us wanting to create some more granular configuration props in the future, but for now, the generic `extra` seems ok to me. I do think it's worth adding a jsdoc above it clarifying what `extra` is and where the result of the function ends up rendering -- just something short like what Ant has in the description you screenshotted.

Reply:
Great call - I added the JSdoc!